src/main/scala/tv/teads/github/api/model/Milestone.scala

```diff
@@ -0,0 +1,69 @@
+package tv.teads.github.api.model
+
+import java.time.ZonedDateTime
+
+import io.circe._, io.circe.syntax._
+import tv.teads.github.api.json.ZonedDateTimeCodec
+
+trait MilestoneCodec {
+  self: UserCodec with LabelCodec with ZonedDateTimeCodec ⇒
+
+  /*implicit lazy val milestoneEncoder = Encoder.instance[Milestone] { milestone ⇒
```

Comment:
I coded it and after realized it was not useful until I code MilestoneService (so it will be used for milestone creation)